IDE/Espressif/ESP-IDF/examples/wolfssl_client/main/client-tls.c

```diff
@@ -42,25 +40,115 @@
     #include <wolfssl/wolfcrypt/mem_track.h>
 #endif
 
-const char *TAG = "tls_client";
+#ifdef WOLFSSL_ESPWROOM32SE
+static const char* TAG = "tls_client";
```

Comment:
Please standardize on `static const char*` for either build type.

Reply:
Done